### PR TITLE
Admin disables merchant

### LINKED ISSUE #85: User Story 48, Admin disables a merchant account
```
As an admin merchant
When I visit the merchant index page
And I click on a "disable" button for an enabled merchant
I am returned to the admin's merchant index page
And I see a flash message that the merchant's account is now disabled
And I see that the merchant's account is now disabled
This merchant cannot log in
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #89

depends on #86

### PR DESCRIPTION
**Pull request by le3ah**
All tests passing; closes #85

### CODE REVIEW

File: spec/features/users/admin/admin_merchant_can_disable_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    it "prevents disabled user from logging in" do
+      @merchant_1 = create(:user, role: 1, enabled: false)
+      visit login_path
+
+      fill_in "email", with: @merchant_1.email
+      fill_in "password", with: @merchant_1.password
+
+      click_on "submit"
+
+      expect(current_path).to eq(login_path)
```
[the remark this answers is not in the document]
Comment (le3ah) [reply spec/features/users/admin/admin_merchant_can_disable_spec.rb]: No, this is the whole story:

[the remark this answers is not in the document]
Comment (le3ah) [reply spec/features/users/admin/admin_merchant_can_disable_spec.rb]: As an admin merchant
When I visit the merchant index page
And I click on a "disable" button for an enabled merchant
I am returned to the admin's merchant index page
And I see a flash message that the merchant's account is now disabled
And I see that the merchant's account is now disabled
This merchant cannot log in

File: app/controllers/admin/merchants_controller.rb [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
         user.switch_enabled
 
         redirect_to admin_merchants_path
-    end 
-  
-end 
\ No newline at end of file
+        if !user.enabled?
+          flash[:disable] = "#{user.name} is now disabled."
```
[the remark this answers is not in the document]
Comment (le3ah) [reply app/controllers/admin/merchants_controller.rb]: Okay, cool.  I fixed it and changed the partial back to what it was.  All tests pass; just pushed it up.

File: spec/features/users/admin/admin_merchant_can_disable_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (le3ah) [reply spec/features/users/admin/admin_merchant_can_disable_spec.rb]: As it stands now, they receive the same "Oh no! Something went wrong" flash message

File: app/controllers/admin/merchants_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (stoic-plus) [reply app/controllers/admin/merchants_controller.rb]: I second what Tony said. `flash[:success]` is meant to be a general _Yay, you did the action and it worked!_

File: spec/features/users/admin/admin_merchant_can_disable_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+      visit admin_merchants_path
+      within "#merchant-#{@merchant_2.id}" do
+        expect(page).to have_button("enable")
+      end
+      within "#merchant-#{@merchant_1.id}" do
+        expect(page).to have_button("disable")
+        click_button "disable"
+      end
+
```
Comment (stoic-plus) [inline spec/features/users/admin/admin_merchant_can_disable_spec.rb]: I would check that the merchant's `enabled` value was changed at the model level here. This may already be covered in some model spec for the `switch_enabled` method but I would still add here to be thorough

Like `expect(@merchant_1.enabled).to be_truthy or eq to true`

Comment (le3ah) [reply spec/features/users/admin/admin_merchant_can_disable_spec.rb]: Not sure why the additional test fails when localhost & save_and_open_page show this as functional.
